builtins/amp-img.js

```diff
@@ -199,8 +199,13 @@ export class AmpImg extends BaseElement {
     if (!this.img_) {
       return;
     }
+    // If the image is server rendered, do not generate sizes.
+    if (this.element.hasAttribute('i-amphtml-ssr')) {
```

Comment:
This is the change I am nervous about: @lannka do AMP Advertisements need to autogenerate `sizes`?

Reply:
Can't speak of other advertisers, but Google Ads server side rendering does not generate srcset or sizes attributes on the img tag.

Reply:
If I'm reading this right, this will mostly disable the sizes-generation for all cached pages? Is that desired?